src/products/eventListing/template.tsx

```diff
@@ -43,8 +43,14 @@ export function EventListing({
   }, [menu, events])
 
   return (
-    <>
-      <div key={id} className="usa-grid usa-grid-full vads-u-padding-bottom--3">
+    <div>
+      <div
+        key={id}
+        className="vads-grid-container vads-u-padding-bottom--3"
+        style={{
+          overflow: 'hidden',
+        }}
```

Comment:
Oh interesting. Why do we need this now? 😕 Did `usa-grid` or `usa-grid-full` have it? 🤔